desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -23,7 +26,7 @@ module.exports = class ArtworkAuctionView extends Backbone.View
     'change .js-artwork-auction-max-bid': 'setMaxBid'
 
   initialize: ({ @data }) -> #
-    setInterval before(10, @updateBidLabel), 1000
+    @updateBidLabel LOT_STANDING_MAX_POLLS
```

Comment:
I want to make sure the first call happens immediately, so that we hopefully get into steady state ASAP, and the user can interact away.